Localization/Resources/es-ES/winget.resw

```diff
@@ -895,6 +903,9 @@ Se pueden configurar mediante el archivo de configuración "winget settings".</v
   <data name="PolicyEnableWinGet" xml:space="preserve">
     <value>Habilitar el Administrador de paquetes de Windows</value>
   </data>
+  <data name="PolicyEnableWindowsPackageManagerCommandLineInterfaces" xml:space="preserve">
+    <value>Habilitar Administrador de paquetes de Windows interfaces de línea de comandos</value>
```

Comment:
We could add a comment here that this means "Enable CLI of/for WPM" so that it gets translated properly